vpr/src/route/rr_graph_generation/clb2clb_directs.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
```

Comment:
Thanks for splitting this code out!
It would be good to add a file scope doxygen comment saying what this file is for / what code should go in it.